page-objects/pages/SubscribePage.js

```diff
@@ -1,47 +1,46 @@
-class Subscribe{
-    get emailAddress() {
-        return $('#MERGE0')
-    }
-
-    get firstName() {
-        return $('#MERGE1')
-    }
-
-    get lastName() {
-        return $('#MERGE2')
-    }
-
-    get submitButton() {
-        return $('.formEmailButton') 
-    }
-
-    get addButton() {
-        return $('.hcard-download')
-    }
-
-    get continueButton() {
-        return $$('.formEmailButton')[0]
-    }
-
-    get cmanageButton() {
-        return $$('.formEmailButton')[1]
-    }
-
-    get emailMeALinkButton(){
-        return $('.formEmailButton')
-    }
-
-    get error() {
-        return $('.formstatus error')
-    }
-
-    email(){
-        return Math.random().toString(36).substring(2,11) + '@domain.com';
-    }
-
-    get confirmSubscribe(){
-        return $('#templateBody')
-    }
-
+class Subscribe {
+  get emailAddress() {
+    return $("#MERGE0");
```

Comment:
I don't think this is a correct selector. It is not suggestive. Maybe you need o find another part from the selector's class.
This comment is valid for all the selectors from this class file.